site2/docs/security-encryption.md

```diff
@@ -35,7 +35,327 @@ If produced messages are consumed across application boundaries, you need to ens
 
 ## Get started
 
-Pulsar encryption allows applications to encrypt messages on the producer side and decrypt messages on the consumer side. See [cookbook](cookbooks-encryption.md) for detailed instructions.
+## Prerequisites
+
+* Pulsar Java/Python/C++/Node.js client 2.7.1 or later versions.
+* Pulsar Go client 0.6.0 or later versions.
+
+## Configure end-to-end encryption
+
+1. Create both public and private key pairs.
+   * ECDSA（for Java and Go clients)
+     ```shell
+     openssl ecparam -name secp521r1 -genkey -param_enc explicit -out test_ecdsa_privkey.pem
+     openssl ec -in test_ecdsa_privkey.pem -pubout -outform pem -out test_ecdsa_pubkey.pem
+     ```
+
+   * RSA (for Python, C++ and Node.js clients)
+     ```shell
+     openssl genrsa -out test_rsa_privkey.pem 2048
+     openssl rsa -in test_rsa_privkey.pem -pubout -outform pkcs8 -out test_rsa_pubkey.pem
```

Comment:
Can we consider using "multiple tabs" like this?

<img width="1171" alt="image" src="https://user-images.githubusercontent.com/50226895/192494366-18e6d5a9-2179-49c8-a5c0-5f581884849c.png">

Reply:
@Anonymitaet Thanks for the suggestion. Will apply this style.